FirstProject/new-evaluation.html

```diff
@@ -0,0 +1,536 @@
+            <div class="col-md-3"><img style="width: 35px; margin-top: 18px;" src="img/2019-200.png"/></div>
+        </div>
+    </div>
+    <div class="container" style="margin-top:100px;">
+        <div class="row">
+            <h2 class="description_section" style="color: cornflowerblue;">Technical level</h2>
+        </div>
+
+        <div class="row" style="background-color: cornflowerblue;">
+                <div class="col-md-3">
+                    <h4 style="color:white;">Trainee</h4>
+                </div>
+
+                <div class="col-md-3">
+                    <h4 style="color: white;">Junior</h4>
+                </div>
+                <div class="col-md-3">
+                    <h4 style="color: white;">Middle</h4>
+                </div>
+                <div class="col-md-3">
+                    <h4 style="color:white;">Senior</h4>
+                </div>
+        </div>
+
+        <div class="row_section" style="border: 1px solid black;">
+            <div class="col-md-3">
+                <ul>
+                    <li class="bullets"><p>&#9711;</p></li>
+                </ul>
+            </div>
+            <div class="col-md-3">
+                <ul style="display: inline-flex;">
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                </ul>
+            </div>
+            <div class="col-md-3">
+                <ul style="display: inline-flex;">
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                </ul>
+            </div>
+            <div class="col-md-3">
+                <ul style="display: inline-flex;">
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                    <li class="bullets"><p>&#9711;</p></li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="row">
+            <h2 class="description_section" style="color: cornflowerblue;">Should the candidate be hired?</h2>
+            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="The type of project that is suitable for the candidate
+Is guidance required for the candidate
+" ></textarea>
+        </div>
+
+        <div class="row">
+            <h2 class="description_section" style="color: cornflowerblue;">General Impression</h2>
+            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="*required
+" ></textarea>
+        </div>
+
+        <div class="row">
+            <h2 class="description_section" style="color: cornflowerblue;">Workflow, Leadership & Soft Skills</h2>
```

Comment:
Hmm... curious as to why the `&` character might be displayed with a red highlight... It seems to indicate an error, but I have no idea what that error might be... :P